themes/Keepcalm-color-theme.json

```diff
@@ -25,7 +25,7 @@
     "statusBarItem.hoverBackground": "#1d191a",
     "statusBarItem.activeBackground": "#191517",
     "editorRuler.foreground": "#2a2d31",
-    "editorLineNumber.foreground": "#3e4651",
+    "editorLineNumber.foreground": "#52627b",
```

Comment:
I don't think this color is aligned with the theme. It looks too cold. Try using an existing color like `#6e7681`, already used for comments, inactive tabs, status bar, etc. I think you'll love it as it is brighter than the current color.

If you like it, update this PR and I'll accept it. It is a bit too bright for me but I can get used to it :-)